manifest.json

```diff
@@ -0,0 +1,38 @@
+{
+  "manifest_version": 3,
+  "name": "Hide Amazon Cart",
+  "version": "0.0.1",
+  "description": "Hide the Amazon cart sidebar while browsing on Amazon",
+   "permissions": ["scripting"],
```

Comment:
After my last project I realized that the tabs permission was probably unnecessary and I was able to remove it without any issues.